### ISSUE TITLE
Library is incompatible with AS 3.0

### ISSUE THREAD

**Issue by stepango** [Commit451/ForegroundViews#10]
Accessing to private resources is now error
```
ERROR: /Users/stepangoncarov/AndroidStudioProjects/90seconds/app/build/intermediates/incremental/mergeDevDebugResources/merged.dir/values/values.xml:731 resource android:attr/foregroundInsidePadding is private
```
on line 
```
<declare-styleable name="ForegroundView"><attr name="android:foreground"/><attr name="android:foregroundInsidePadding"/><attr name="android:foregroundGravity"/></declare-styleable>
```

**Comment by stepango** [comment]
As a workaround my fork could be used compile 'com.github.stepango:foregroundviews:master-SNAPSHOT'

**Comment by Jawnnypoo** [comment]
Thanks for reporting, fixed in `2.4.0`
